server/sonar-server/src/test/java/org/sonar/server/computation/step/ReportComputeMeasureVariationsStepTest.java

```diff
@@ -226,6 +221,28 @@ public void set_variation_on_all_numeric_metrics() {
     assertThat(measureRepository.getRawMeasure(PROJECT, BUILD_BREAKER_METRIC).get().getVariations().getVariation1()).isEqualTo(-1d);
   }
 
+  @Test
+  public void do_not_set_variations_on_numeric_metric_for_developer() {
```

Comment:
I did something of the like.
thank you a lot for spotting the test was crap, I also improved production code